src/webapp/components/survey/hook/useSurveyForm.ts

```diff
@@ -151,6 +156,21 @@ export function useSurveyForm(formType: SURVEY_FORM_TYPES, eventId: string | und
         currentCaseReportForm?.id,
     ]);
 
+    useEffect(() => {
+        if (prevalenceCurrentFormId) {
+            compositionRoot.surveys.getSurveyAntibioticsBlacklist
```

Comment:
this call getSurveyAntibioticsBlacklist is not required. You can get the antibioticslist by

```
 const antibioticsBlacklist = currentModule?.rulesBySurvey?.find(
                rule => rule.surveyId === prevalenceCurrentFormId
            )?.antibioticBlacklist;
```
Please remove the usecase and repository functions which wont be used . Also this state is not required because you can calculate the antibioticslist inside useASTGuidelinesOptions()::getAntibioticOptions